currency_rate_update_xe/tests/test_currency_rate_update_xe.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestResCurrencyRateProviderXE(common.TransactionCase):
+    def setUp(self):
```

Comment:
You shouldn't change from `setUpClass` to `setUp`, as it's less efficient.

Reply:
Fixed please review